hack/e2e/eksctl.sh

```diff
@@ -68,14 +63,14 @@ function eksctl_create_cluster() {
 
   if [[ "$WINDOWS" == true ]]; then
```

Comment:
Nit : We do not support windows, but this is a day one issue, might want to remove this in a separate PR later

Reply:
Sure